services/workflows-service/src/alert/alert.service.ts

```diff
@@ -152,10 +168,10 @@ export class AlertService {
       async (executionRow: Record<string, unknown>) => {
         try {
           if (
-            _.some(
-              inlineRule.subjects,
-              field => _.isNull(executionRow[field]) || _.isUndefined(executionRow[field]),
-            )
+            _.some(inlineRule.subjects, field => {
+              const val = findValNoCase(executionRow, field);
+              _.isNull(val) || _.isUndefined(val);
+            })
```

Comment:
Executing multiline functions inside `if` statement is hard to read. Consider extracting it into a variable.